### PR TITLE
Add Django Example

### PR DESCRIPTION
**Pull request by styfle**
Try it here: https://python-django.now-examples.now.sh/

### CODE REVIEW

**Comment by styfle** [comment]
cc @redreceipt to see if there is a way to set django config without the environment variable

**Comment by redreceipt** [comment]
@styfle which django config are you referring to here?

**Comment by sophearak** [comment]
I managed to get it working. I have to modified route and some setting to get it working.
My guess the way `django` handle routing are slightly different from `flask`.

Try it here: https://python-django-1kz3fkvui.now.sh

cc @styfle

**Comment by styfle** [comment]
@redreceipt I meant the settings. The docs mention there is a way to assign settings without the environment variable but my attempt was unsuccessful.

https://docs.djangoproject.com/en/2.2/topics/settings/#using-settings-without-setting-django-settings-module

**Comment by redreceipt** [comment]
@styfle ok I understand what's going on now, you have a settings module for the project (python-Django) *and* a settings module for the about page which is a Django app. I'm not sure if this lines up with Django best practices. I believe there should be only one `settings.py` per project. My fear is that it will create problems down the road with no help to fall back on. Users making Django apps are not going to know how your example was pieced together.

That being said, I think that may explain why we're having difficulty importing the `about.settings`.

File: python-django/urls.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from django.http import HttpResponse
+
+source = 'https://github.com/zeit/now-examples/tree/master/python-django'
+css = '<link rel="stylesheet" href="/css/style.css" />'
+
+def index(request):
+    return HttpResponse("%s<h1>Django on Now 2.0</h1><p>You are viewing a Django application written in Python running on Now 2.0.</p><p>Visit the <a href='./about'>about</a> page or view the <a href='%s'>source code</a>.</p>" % (css, source), content_type='text/html')
+
+def about(request):
+    return HttpResponse("%s<h1>About</h1><p>You are viewing a Django.</p><p>Visit the <a href='./'>home</a> page.</p>" % (css), content_type='text/html')
```
Comment (styfle) [inline python-django/urls.py]:
```suggestion
    return HttpResponse("%s<h1>About</h1><p>You are viewing a Django application.</p><p>Visit the <a href='./'>home</a> page.</p>" % (css), content_type='text/html')
```

**Comment by styfle** [comment]
> I'm not sure if this lines up with Django best practices. I believe there should be only one settings.py per project.

@redreceipt After some consideration, I think we agree. Django is not very suitable for serverless and trying to deviate too far from the recommended path makes the framework moot.

This example has been updated to follow Django best practices with comments for Now 2.0 best practices (such as static CDN).

**Pull request merged**